main/src/main/java/cgeo/geocaching/MainActivity.java

```diff
@@ -155,7 +155,7 @@ private void fillView(final View connectorInfo, final ILogin conn) {
                             connectorStatus.setOnClickListener(connectorConfig);
 
                             final Button manualLogin = connectorInfo.findViewById(R.id.manual_login);
-                            manualLogin.setVisibility(!conn.isLoggedIn() && !isLoggingIn && !isLoggingOk && conn.supportsManualLogin() && Settings.getGcCredentials().isValid() ? View.VISIBLE : View.GONE);
+                            manualLogin.setVisibility(connInfo.toString().contains("Failed to log in to geocaching.com due to captcha required") ? View.VISIBLE : View.GONE);
```

Comment:
Does this work for non-english installations?

Reply:
Never mind, I just saw your new issue about error messages being untranslated :-)